build.rs

```diff
@@ -189,11 +189,13 @@ fn create_ss58_registry(json: &str) -> Result<proc_macro2::TokenStream, String>
 
 		impl Ss58AddressFormat {
 			/// Network/AddressType is reserved for future use.
+			#[must_use]
```

Comment:
```suggestion
```

Reply:
What is the reason for these `must_use` statements?

Please remove all of them.